Scribe/ParentTableCellModel.swift

```diff
@@ -38,16 +38,16 @@ struct Section {
   init(
     sectionTitle: String,
     imageString: String? = nil,
-    hasToggle: Bool = false, h _: Bool = false,
-
+    hasToggle: Bool = false, 
+    hasNestedNavigation: Bool = false,
```

Comment:
Honestly very confused how this even got through 🤔 Maybe just something from auto save when I was using VS Code to format? Thanks again! :)